base/src/main/java/com/smartdevicelink/managers/screen/BaseScreenManager.java

```diff
@@ -377,6 +378,21 @@ public String getTitle(){
 		return this.textAndGraphicManager.getTitle();
 	}
 
+	/**
+	 * Change the current layout to a new layout and optionally update the layout's night and day color schemes. The values set for the text, graphics,
+	 * buttons and template title persist between layout changes. To update the text, graphics, buttons and template title at the same time as the template,
+	 * batch all the updates between `beginUpdates` and `endUpdates`. If the layout update fails while batching, then the updated text, graphics, buttons or template title will also not be updated.
+	 *
+	 * If you are connected on a < v6.0 connection and batching the update, the layout will be updated, then the text and graphics will be updated.
+	 * If you are connected on a >= v6.0 connection, the layout will be updated at the same time that the text and graphics are updated.
+	 *
+	 * @param templateConfiguration The new configuration of the template, including the layout and color scheme.
+	 * @param listener A listener that will be called when the layout change finished.
+	 */
+	public void changeLayout(TemplateConfiguration templateConfiguration, CompletionListener listener) {
+		textAndGraphicManager.changeLayout(templateConfiguration, listener);
```

Comment:
While testing, I noticed that the listener doesn't get called if the developer is using the batching mode

Reply:
we did this because if it's batched and that batched update gets overridden, it will return false for the changeLayout. This is keeping in line with how the TextAndGraphicsManager works with overriding updates.